pyocd/target/builtin/target_MIMXRT1015xxxxx.py

```diff
@@ -0,0 +1,745 @@
+    0xbf00bf00, 0xbf00bf00, 0xf3bfbf00, 0xbf008f6f, 0xbf00bf00, 0x69404836, 0x04492101, 0x49344308,
+    0xbf006148, 0xbf00bf00, 0x8f4ff3bf, 0xbf00bf00, 0xbf00bf00, 0xbf00bf00, 0x8f6ff3bf, 0xbf00bf00,
+    0x4770bf00, 0x2300b510, 0x34804c29, 0xbf006063, 0xbf00bf00, 0x8f4ff3bf, 0xbf00bf00, 0x4623bf00,
+    0x4b2c6818, 0x0b594003, 0x4b2bbf00, 0x08da4003, 0x014bbf00, 0x40234c29, 0x0fa40794, 0x432307a4,
+    0x62234c23, 0xbf00bf00, 0x1e524613, 0xd1f02b00, 0x1e49460b, 0xd1e82b00, 0xbf00bf00, 0xf3bfbf00,
+    0xbf008f4f, 0xbf00bf00, 0x695b4b11, 0x43234c13, 0x61634c0f, 0xbf00bf00, 0xf3bfbf00, 0xbf008f4f,
+    0xbf00bf00, 0xbf00bf00, 0xf3bfbf00, 0xbf008f6f, 0xbf00bf00, 0x0000bd10, 0x400b8000, 0x400d0000,
+    0xd928c520, 0x400bc000, 0x0000ffff, 0xe000e000, 0xe000ed00, 0x400d8000, 0x016e3600, 0x00010000,
+    0x400fc000, 0x400d8100, 0x1dcd6500, 0x0000020c, 0xe000ef40, 0x0fffe000, 0x00001ff8, 0x00003fe0,
+    0x4605b5fe, 0x460c4610, 0xd0734318, 0x468c46ae, 0x1aad2000, 0x419c4601, 0x4666d367, 0x24012700,
+    0x1ab6463d, 0xd302419d, 0x463a4613, 0x46652421, 0x042f4676, 0x433e0c36, 0x1ab60c2d, 0xd304419d,
+    0x041b0c15, 0x0412432b, 0x46653410, 0x062f4676, 0x433e0a36, 0x1ab60a2d, 0xd304419d, 0x021b0e15,
+    0x0212432b, 0x46653408, 0x072f4676, 0x433e0936, 0x1ab6092d, 0xd304419d, 0x011b0f15, 0x0112432b,
+    0x46651d24, 0x07af4676, 0x433e08b6, 0x1ab608ad, 0xd304419d, 0x009b0f95, 0x0092432b, 0x46651ca4,
+    0x07ef4676, 0x433e0876, 0x1ab6086d, 0xd31a419d, 0x415b1892, 0xe0161c64, 0x46761800, 0x41494665,
+    0x419d1ab7, 0x90009101, 0x4660d309, 0x41981ab1, 0x4684468e, 0x99019800, 0x1c402500, 0x07dd4169,
+    0x432a0852, 0x1e64085b, 0x4672d5e6, 0xb0034663, 0xe7ffbdf0, 0x46012000, 0x46c046c0, 0x4623462a,
+    0x0000e7f5, 0x4604b50e, 0x460e4615, 0xf000a012, 0x4620f93b, 0xf938f000, 0xf000a015, 0x4630f935,
+    0xf932f000, 0xf000a014, 0x466cf92f, 0x72e02000, 0x72a0200a, 0xe007340a, 0x4628210a, 0xf877f000,
+    0x46053130, 0x70211e64, 0xdcf52d00, 0xf0004620, 0xf000f91b, 0x0000f90f, 0x202a2a2a, 0x65737361,
+    0x6f697472, 0x6166206e, 0x64656c69, 0x0000203a, 0x6966202c, 0x0020656c, 0x696c202c, 0x0020656e,
+    0xb5104603, 0x079b430b, 0x2a04d10f, 0xc808d30d, 0x1f12c910, 0xd0f842a3, 0xba21ba18, 0xd9014288,
+    0xbd102001, 0x43c02000, 0x2a00bd10, 0x07d3d003, 0x1c52d003, 0x2000e007, 0x7803bd10, 0x1c40780c,
+    0x1b1b1c49, 0x7803d107, 0x1c40780c, 0x1b1b1c49, 0x1e92d101, 0x4618d1f1, 0xe001bd10, 0x1f09c004,
+    0xd2fb2904, 0xd501078b, 0x1c808002, 0xd00007c9, 0x47707002, 0xd00b2900, 0xd00207c3, 0x1c407002,
+    0x29021e49, 0x0783d304, 0x8002d502, 0x1e891c80, 0x2200e7e3, 0x2200e7ee, 0x2200e7df, 0x428b0903,
+    0x0a03d32c, 0xd311428b, 0x469c2300, 0x4603e04e, 0xd43c430b, 0x08432200, 0xd331428b, 0x428b0903,
+    0x0a03d31c, 0xd301428b, 0xe03f4694, 0x428b09c3, 0x01cbd301, 0x41521ac0, 0x428b0983, 0x018bd301,
+    0x41521ac0, 0x428b0943, 0x014bd301, 0x41521ac0, 0x428b0903, 0x010bd301, 0x41521ac0, 0x428b08c3,
+    0x00cbd301, 0x41521ac0, 0x428b0883, 0x008bd301, 0x41521ac0, 0x428
```

Comment:
The comment here (and on RT1011) has the wrong memory size, I think. Not a big deal since it doesn't affect functionality, but good to have it match the hardware.

Reply:
Thanks, @flit , updated.